### ISSUE TITLE
look into supporting Android Studio 3.1 Canary

### ISSUE THREAD

**Issue by devoncarew** [flutter/flutter-intellij#1650]
As the canary release matures, we should look into supporting it.

**Comment by devoncarew** [comment]
@stevemessick, fyi, I'm moving this into M23 (next month's milestone); let me know if that sounds reasonable.

**Comment by stevemessick** [comment]
I don't believe we have (or will have) the resources to track the canary releases. I think it would be better to commit to supporting the beta channel.

AS 3.1 beta 1 is out now and we should be working on support for it.

**Comment by stevemessick** [comment]
There are many bugs referenced here that are marked as closed. They are all due to load errors, which are fixed in https://github.com/flutter/flutter-intellij/pull/1792.
